src/Wpf.Ui/Controls/Arc/Arc.cs

```diff
@@ -3,27 +3,25 @@
 // Copyright (C) Leszek Pomianowski and WPF UI Contributors.
 // All Rights Reserved.
 
+using System.Windows.Controls;
+using System.Windows.Shapes;
 using Point = System.Windows.Point;
 using Size = System.Windows.Size;
+#pragma warning disable SA1124
+#pragma warning disable CS0108
 
-// ReSharper disable once CheckNamespace
 namespace Wpf.Ui.Controls;
 
-/// <summary>
```

Comment:
Why removing docs?

Reply:
Well i removed them for a better overview and just forgot to add them back x.x